.asf.yaml

```diff
@@ -31,6 +33,16 @@ github:
     squash:  true
     merge:   true
     rebase:  true
+  # no force push to listed branches
+  protected_branches:
+    master:
+      required_status_checks:
+          # strict means "Require branches to be up to date before merging".
+          strict: false
+          # list of names of PR checks that must pass
+          contexts:
+            - CI_GREEN
```

Comment:
this should give up force push protection for master and enable the status check requirement which is represented by the CI_GREEN job.

I am scared to add travis there. If something doesn't work or the check is not recognized for some reason we lock ourself out.